apps/meteor/client/views/room/body/hooks/useFileUploadDropTarget.ts

```diff
@@ -55,7 +55,8 @@ export const useFileUploadDropTarget = (): readonly [
 		const uniqueFiles = getUniqueFiles();
 
 		const uploads = Array.from(uniqueFiles).map((file) => {
-			Object.defineProperty(file, 'type', { value: mime.lookup(file.name) });
+			const fileMimeType = mime.lookup(file.name);
+			Object.defineProperty(file, 'type', { value: typeof fileMimeType === 'string' ? fileMimeType : 'application/octet-stream' });
```

Comment:
Wouldn't it be better to create a helper function? It seems it's being used in multiple places...

Reply:
Good idea, done!